js/shutterbug-worker.js

```diff
@@ -12,6 +15,16 @@ function getID () {
   return _id++
 }
 
+function getURLParam(name) {
+  const url = (self || window).location.href;
+  name = name.replace(/[[]]/g, "\\$&");
+  const regex = new RegExp(`[#?&]${name}(=([^&#]*)|&|#|$)`);
+  const results = regex.exec(url);
+  if (!results) return null;
+  if (!results[2]) return true;
+  return decodeURIComponent(results[2].replace(/\+/g, " "));
+}
```

Comment:
FYI, the [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) API is another alternative that doesn't require an external library.

Reply:
Thanks, I didn't know about it. It also seems to be widely supported (just missing IE). I've just replaced the regexp with it.